drain3/template_miner.py

```diff
@@ -23,6 +24,7 @@
 
 config_filename = 'drain3.ini'
 
+ParameterDetails = NamedTuple("ParameterDetails", [("value", str), ("mask_name", str)])
```

Comment:
I suggest calling this `ExtractedParameter`

Reply:
This class could also be used in case parameter extraction during template mining/matching is ever implemented.
I think extracted parameter would be misnamed in that case.
What do you think?

Reply:
Why? its also an extracted parameter in this case.

Reply:
Fair point!